src/Arcus.EventGrid.EventHubs/Contracts/Events/v1/CaptureFileCreated.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Arcus.EventGrid.Contracts;
+using Arcus.EventGrid.EventHubs.Contracts.Events.v1.Data;
+
+namespace Arcus.EventGrid.EventHubs.Contracts.Events.v1
+{
+    public class CaptureFileCreated : Event<EventHubCaptureEventData>
+    {
+        public CaptureFileCreated()
+        {
+        }
+
+        public CaptureFileCreated(string id) : base(id)
```

Comment:
Are these valid if there `null` ? If not, should we guard against it?

Reply:
Problem is there's a default constructor already, which allows the consumer to create something with null id's etc, I would assume?

Reply:
The base constructor is guarding for empty strings but the default constructors are there for serialization indeed. Not sure if this will work but can try to make them `protected`

Reply:
Ah, there exists some other event types that also needs guarding I think?